microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -91,3 +91,39 @@ groups:
         title: "Metadata Overhead Above 90% on Bucket: {{ $labels.bucket }}, Node: {{ $labels.instance }}"
         description: The percentage of memory that is taken up by metadata is over 90%
         remediation: Increase memory allocation for bucket or change the evictionPolicy of the bucket from `Value-only` (be aware this will have an adverse effect on performance).
+
+    - alert: CB90061-diskWriteQueueLength-Alert
+      expr: |
+        kv_ep_storage_age_seconds > 99
+      for: 1m
+      labels:
+        job: couchbase_prometheus
+        kind: bucket
+        health_check_id: CB90061
+        health_check_name: diskWriteQueueLength
+        cluster: '{{ $labels.cluster }}'
+        bucket: '{{ $labels.bucket }}'
+        severity: critical
+      annotations:
+        title: "Long disk write queue on Bucket: {{ $labels.bucket }}"
+        description: The age of items in the disk write queue is over 100 seconds.
+        remediation: Review your hardware for malfunctions or sizing issues. 
+          If the problem persists, then please contact Couchbase Technical Support. 
+
+    - alert: CB90061-diskWriteQueueLength-Warn
+      expr: |
+        (kv_ep_storage_age_seconds > 50) and (kv_ep_storage_age_seconds < 100)
```

Comment:
Might be able to simplify this
![image](https://user-images.githubusercontent.com/2904440/151366348-bcee70f0-ec39-4127-989f-bb52e3136eac.png)